src/scss/_components.respondents.scss

```diff
@@ -0,0 +1,364 @@
+.cmp-respondents {
+  width: calc(100% - 2rem);
+  max-width: 74rem;
+  margin: 4rem auto 6rem;
+
+  @media (min-width: $bp-respond-medium-layout) {
+    margin: 6rem auto 8rem;
+  }
+
+  &__list {
+    @include delist;
+    margin-top: 4rem;
+
+    @supports (display: grid) {
+      display: grid;
+      grid-row-gap: 6rem;
+    }
+
+    @media (min-width: $bp-respond-start-grid) {
+      display: flex;
+      flex-wrap: wrap;
+
+      @supports (display: grid) {
+        display: grid;
+        grid-template: 1fr 1fr / 1fr 1fr;
+        grid-column-gap: 2rem;
+        grid-row-gap: 4rem;
```

Comment:
At this row gap, the pyramid and diamond get really close together at one point (saw this in Chrome, didn't check other browsers):

<img width="230" alt="Screen Shot 2019-06-28 at 9 59 22 AM" src="https://user-images.githubusercontent.com/23404383/60347507-7f7e5380-998b-11e9-8651-18618e0478e3.png">

Changing that `grid-row-gap` to 5rem or 6rem should help.